frontend/src/components/messages/NewRequestDialog.vue

```diff
@@ -230,6 +230,18 @@
                 :disabled="isLoadingOrDisabled"></v-textarea>
             </v-col>
           </v-row>
+
+          <v-row v-if="isAnIrregularExpenseRequest" class="my-3">
+            <v-col cols="12">
+              <h5>Download a Irregular Expense Form then fill it out!</h5>
+            </v-col>
+            <v-row>
+              <v-col cols="12" class="ml-3">
+                <AppButton id="download-irregular-expense-form" size="large" width="200px" :disabled="isDisabled" :loading="isLoading">Download Form</AppButton>
```

Comment:
My understanding is that this would be a hyperlink to an externally hosted site.